src/apps/deposit/components/SendAsset/SendAsset.tsx

```diff
@@ -156,14 +176,18 @@ const SendAsset = () => {
               />
             )}
           </div>
-          {Number(amount) > 0 && (
+          {Number(amount) > 0 && nftType !== 'ERC1155' ? (
             <button
               type="button"
               onClick={handleSendTx}
               className="w-fit h-fit self-center px-4 py-2 mt-4 rounded-md bg-purple_medium hover:bg-purple_light"
             >
               Start transfer
             </button>
+          ) : (
+            <p className="text-base">
+              We do not currently support ERC1155 deposit.
```

Comment:
```suggestion
              Sorry, we do not currently support ERC1155 transfers. You can manually transfer this NFT to your PillarX wallet address: {walletAddress}
```

Remember to always help the user continue their journey when hitting a dead-end like this 🙏